<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bcBlackCatEditorClass</class>
 <widget class="QMainWindow" name="bcBlackCatEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>1200</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BlackCatEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="leftMargin">
     <number>3</number>
    </property>
    <property name="topMargin">
     <number>3</number>
    </property>
    <property name="rightMargin">
     <number>3</number>
    </property>
    <property name="bottomMargin">
     <number>3</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QFrame" name="leftFrame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QSplitter" name="splitter">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <widget class="QFrame" name="leftRenderFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>400</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
           </widget>
           <widget class="QFrame" name="leftBottomFrame">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="leftMargin">
              <number>3</number>
             </property>
             <property name="topMargin">
              <number>3</number>
             </property>
             <property name="rightMargin">
              <number>3</number>
             </property>
             <property name="bottomMargin">
              <number>3</number>
             </property>
             <item>
              <widget class="QTabWidget" name="leftBottomTab">
               <property name="currentIndex">
                <number>0</number>
               </property>
               <widget class="QWidget" name="consoleTab">
                <attribute name="title">
                 <string>Console</string>
                </attribute>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="rightFrame">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>360</width>
          <height>0</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>10</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QToolBox" name="rightToolBox">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <property name="fontAwesome" stdset="0">
            <string>handpointero,areachart</string>
           </property>
           <widget class="QWidget" name="rightToolBoxObjectSelect">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>338</width>
              <height>736</height>
             </rect>
            </property>
            <attribute name="label">
             <string>ObjectSelection</string>
            </attribute>
            <widget class="QLabel" name="entityNameLabel">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>10</y>
               <width>81</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string>EntityName:</string>
             </property>
            </widget>
            <widget class="QLabel" name="entityNameLabelValue">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>10</y>
               <width>231</width>
               <height>16</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </widget>
           <widget class="QWidget" name="rightToolBoxTerrain">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>338</width>
              <height>736</height>
             </rect>
            </property>
            <attribute name="icon">
             <iconset theme="areaChart">
              <normaloff>.</normaloff>.</iconset>
            </attribute>
            <attribute name="label">
             <string>Terrain</string>
            </attribute>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="terrainHeightLable">
               <property name="text">
                <string>Height: </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="terrainHeightValue">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QSlider" name="terrainHeightSlider">
               <property name="minimum">
                <number>-20</number>
               </property>
               <property name="maximum">
                <number>20</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="pageStep">
                <number>2</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="tickInterval">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="terrainSmoothLable">
               <property name="text">
                <string>Smooth:</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0" colspan="2">
              <widget class="QSlider" name="terrainSmoothSlider">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>5</number>
               </property>
               <property name="pageStep">
                <number>1</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLabel" name="terrainSmoothValue">
               <property name="text">
                <string>1</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0">
              <widget class="QLabel" name="terrainMaterialLable">
               <property name="text">
                <string>Material: </string>
               </property>
              </widget>
             </item>
             <item row="11" column="0" colspan="2">
              <widget class="QComboBox" name="terrainMaterialCombo">
               <item>
                <property name="text">
                 <string>Material 1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Material 2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Material 3</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Material 4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Material 5</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Material 6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Material 7</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Material 8</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QSlider" name="terrainRadiusSlider">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>50</number>
               </property>
               <property name="singleStep">
                <number>1</number>
               </property>
               <property name="value">
                <number>25</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="tickInterval">
                <number>0</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="terrainRadiusLable">
               <property name="text">
                <string>Radius: </string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QLabel" name="terrainRadiusValue">
               <property name="text">
                <string>25</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="toolsDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>55</width>
     <height>914</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>55</width>
     <height>524287</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::NoDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string notr="true"/>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="toolsDockContents">
    <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>5</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>5</number>
     </property>
     <property name="bottomMargin">
      <number>600</number>
     </property>
     <item>
      <widget class="QToolButton" name="objectSelectButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Object selection</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="fontAwesome" stdset="0">
        <string>handpointero</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="terrainHeightButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Terrain Height</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="fontAwesome" stdset="0">
        <string>areachart</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="terrainSmoothButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Terrain Smooth</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="fontAwesome" stdset="0">
        <string>bullseye</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="terrainMaterialButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Terrain Material</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="fontAwesome" stdset="0">
        <string>envira</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="terrainMaterialSmoothButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Terrain Material Smooth</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
       <property name="fontAwesome" stdset="0">
        <string>bullseye</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bcBlackCatEditor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>terrainHeightSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>terrainHeightValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1572</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>1336</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terrainSmoothSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>terrainSmoothValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1572</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>1572</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>terrainRadiusSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>terrainRadiusValue</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1281</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>1317</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_pushButton_clicked()</slot>
 </slots>
</ui>
